R/summary_funs.R

```diff
@@ -88,23 +88,31 @@ weighted_summary_means <- function(y_wobs_test, family, wdraws, mu, dis, cl_ref,
 # statistics relative to the baseline model of that size (`nfeat_baseline = Inf`
 # means that the baseline model is the reference model).
 .tabulate_stats <- function(varsel, stats, alpha = 0.05,
-                            nfeat_baseline = NULL, resp_oscale = TRUE, ...) {
+                            nfeat_baseline = Inf, resp_oscale = TRUE, ...) {
   stat_tab <- data.frame()
-  summ_ref <- varsel$summaries$ref
-  summ_sub <- varsel$summaries$sub
+  summaries_ref <- varsel$summaries$ref
+  summaries_sub <- varsel$summaries$sub
+  if (!is.null(varsel$summaries_fast)) {
+    summaries_fast_sub <- varsel$summaries_fast$sub
+    if (any(stats %in% c("auc"))) {
+      warning("Subsampling LOO with AUC not implemented. Using fast LOO for submodel AUC.")
```

Comment:
I think we should point this out in the NEWS, in the docs, etc. because users might build a quite complicated and costly `cv_varsel()` call with subsampled LOO-CV, thinking they would be able to use its results for AUC-based model selection, only to be notified later during post-processing (when calling `plot.vsel()` or `summary.vsel()`) that AUC does not work for subsampled LOO-CV. So it's better to point this out early, I guess.